packages/pyright-internal/src/analyzer/backgroundAnalysisProgram.ts

```diff
@@ -18,7 +18,16 @@ import { Range } from '../common/textRange';
 import { AnalysisCompleteCallback, analyzeProgram } from './analysis';
 import { CacheManager } from './cacheManager';
 import { ImportResolver } from './importResolver';
-import { Indices, MaxAnalysisTime, OpenFileOptions, Program } from './program';
+import { MaxAnalysisTime, OpenFileOptions, Program } from './program';
+
+export enum InvalidatedReason {
+    Reanalyzed,
+
+    SourceWatcherChanged,
+
+    LibraryWatcherChanged,
+    LibraryWatcherContentOnlyChanged,
+}
```

Comment:
The formatting of this enum seems inconsistent with existing enums in Pyright.
```suggestion
export enum InvalidatedReason {
    Reanalyzed,
    SourceWatcherChanged,
    LibraryWatcherChanged,
    LibraryWatcherContentOnlyChanged,
}
```